test/gtest/cfg.cpp

```diff
@@ -255,3 +255,42 @@ End
 
   EXPECT_EQ(ss.str(), analyzerText);
 }
+
+TEST_F(CFGTest, FinitePowersetLatticeFunctioning) {
+
+  std::vector<std::string> initialSet = {"a", "b", "c", "d", "e", "f"};
+  FinitePowersetLattice<std::string> lattice(std::move(initialSet));
+
+  FinitePowersetLattice<std::string>::Element element1 = lattice.getBottom();
```

Comment:
No need to write out such a verbose type.
```suggestion
  auto element1 = lattice.getBottom();
```